src/Common/src/Interop/Unix/libssl/SslConnectionInfo.cs

```diff
@@ -26,7 +26,8 @@ internal SslConnectionInfo(SafeSslHandle sslContext)
                 out DataCipherAlg,
                 out KeyExchangeAlg,
                 out DataHashAlg,
-                out DataKeySize))
+                out DataKeySize,
+                out DataHashKeySize))
             {
                 throw Interop.OpenSsl.CreateSslException(SR.net_ssl_get_connection_info_failed);
             }
```

Comment:
seems this todo we cannot proceed in , we have already done the investigations around. So just making a note for this. No point of opening another TODO. 
Will delete this TODO comment.